staking_deposit/cli/existing_mnemonic.py

```diff
@@ -58,7 +58,6 @@ def validate_mnemonic(ctx: click.Context, param: Any, mnemonic: str) -> str:
     callback=captive_prompt_callback(
         lambda num: validate_int_range(num, 0, 2**32),
         lambda: load_text(['arg_validator_start_index', 'prompt'], func='existing_mnemonic'),
-        lambda: load_text(['arg_validator_start_index', 'confirm'], func='existing_mnemonic'),
```

Comment:
I think we would need to add confirmation for both cases (with and without the file.)
It's fine to remove it from here.